sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -273,18 +313,37 @@ def test_predict_transform_copy():
     assert_true(np.all(X.mean(axis=0) != 0))
 
 
-def test_scale():
-    d = load_linnerud()
-    X = d.data
-    Y = d.target
+def test_scale_and_stability():
+    # We test scale=True parameter
+    # This allows to check numerical stability over platforms as well
 
+    d = load_linnerud()
+    X1 = d.data
+    Y1 = d.target
     # causes X[:, -1].std() to be zero
-    X[:, -1] = 1.0
-
-    for clf in [pls_.PLSCanonical(), pls_.PLSRegression(),
-                pls_.PLSSVD()]:
-        clf.set_params(scale=True)
-        clf.fit(X, Y)
+    X1[:, -1] = 1.0
+
+    # Test with X_score[:, 1] == 0, Y_score[:,1] == 0
+    X2 = np.array([[0., 0., 1.], [1., 0., 0.], [2., 2., 2.], [3., 5., 4.]])
```

Comment:
putting one row per line would make this array more readable. Where do the values come from?